files/ubuntu-18.04/sge_preinstall.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -e
+echo "Downloading and extracting source packages for $dest_package"
+
+orig_package='gridengine_8.1.9+dfsg'
```

Comment:
is it possible to change the cwd of this script to something temporary and not "#{node['cfncluster']['sources_dir']}"?

Reply:
OK